packages/wallet/backend/src/transaction/service.ts

```diff
@@ -90,4 +93,31 @@ export class TransactionService implements ITransactionService {
 
     return transactions
   }
+
+  async processPendingIncomingPayments(): Promise<string | undefined> {
+    return this.deps.knex.transaction(async (trx) => {
+      const now = new Date(Date.now())
+      const [transaction] = await Transaction.query(trx)
```

Comment:
can all expired transactions be updated at the same time?

Reply:
I don't really like the idea of mass-updating the transactions. 🤔